tekton/ci/interceptors/add-team-members/cmd/add-team-members/main.go

```diff
@@ -102,24 +102,28 @@ func makeAddTeamMembersHandler(orgMembersFetcher, teamMembersFetcher urlToList,
 			marshalError(err, w)
 			return
 		}
-		// Add the PR body to the original body
+		// Add the org members to the original body
 		jsonBody[RootExtensionsKey].(map[string]interface{})[prExtensionsKey].(map[string]interface{})[orgMembersKey] = orgMembers
-		// Get the team URL from the body
-		teamUrl, err := getTeamUrl(jsonBody)
-		if err != nil {
-			log.Printf("failed to extract the Team URL from the body: %q", err)
-			marshalError(err, w)
-			return
-		}
-		// Get the list of org members from the URL
-		teamMembers, err := teamMembersFetcher(teamUrl, token)
-		if err != nil {
-			log.Printf("failed to get the list of team members: %q", err)
-			marshalError(err, w)
-			return
+
+		_, ok := jsonBody[teamKey]
+		if ok {
```

Comment:
Should we add a log message if this was `ok`? Might make debugging easier in the future. But I know that logs in the current codebase are a bit sparse.

Reply:
Current approach in the codebase seems to be logging only on failure, but I think this could help. Let's see what the `OWNERS` think.

Reply:
Heh, this interceptor is very simple, I didn't even setup a logging framework.
It's ok to add logging for ok cases too, as long they include info relevant for debugging.